src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -67,145 +67,66 @@ class ItemPage extends React.Component {
 
-      // external links
-      if (f.url) {
-        detailFields.push({
-          term: f.label,
-          definition: (
-            <ul>
-              {record[f.field].map((value, i) => {
-                const v = f.field === 'idOclc' ? 'View in Worldcat' : value;
-                return <li key={i}><a target="_blank" title={v} href={f.url(v)}>{v}</a></li>;
-              })}
-            </ul>
-          ),
-        });
-
       // list of links
-      } else if (fieldValue['@id']) {
-        detailFields.push({
-          term: f.label,
-          definition: (
-            <ul>{record[f.field].map((obj, i) => (
-              <li key={i}>
-                <Link
-                  onClick={e => this.onClick(e, `filters[${f.field}]=${obj['@id']}`)}
-                  title={`Make a new search for ${f.label}: ${obj.prefLabel}`}
-                  to={`/search?q=${encodeURIComponent(`filters[${f.field}]=${obj['@id']}`)}`}
-                >{obj.prefLabel}</Link>
-              </li>))}
-            </ul>),
-        });
+      if (fieldValue['@id']) {
+        itemDetails.push({ term: f.label, definition: <ul>{record[f.field].map((obj, i) => (<li key={i}><a onClick={(e) => this.onClick(e, `${f.field}:"${obj['@id']}"`)} href={`/search?q=${encodeURIComponent(`${f.field}:"${obj['@id']}"`)}`}>{obj.prefLabel}</a></li>))}</ul> });
+
       // list of links
       } else if (f.linkable) {
-        if (f.field === 'contributorLiteral') {
-          detailFields.push({
-            term: f.label,
-            definition: record[f.field].map((value, i) => {
-              const comma = record[f.field].length > 1 ? ', ' : ' ';
-              return (
-                <span key={i}>
-                  <Link
-                    onClick={e => this.onClick(e, `filters[${f.field}]=${value}`)}
-                    title={`Make a new search for ${f.label}: "${value}"`}
-                    to={`/search?q=${encodeURIComponent(`filters[${f.field}]=${value}`)}`}
-                  >
-                    {value}
-                  </Link>{comma}
-                </span>
-              );
-            }),
-          });
-        } else {
-          detailFields.push({
-            term: f.label,
-            definition: (
-              <ul>{record[f.field].map((value, i) => (
-                <li key={i}>
-                  <Link
-                    onClick={e => this.onClick(e, `filters[${f.field}]=${value}`)}
-                    title={`Make a new search for ${f.label}: "${value}"`}
-                    to={`/search?q=${encodeURIComponent(`filters[${f.field}]=${value}`)}`}
-                  >{value}</Link>
-                </li>))}
-              </ul>),
-          });
-        }
+        itemDetails.push({ term: f.label, definition: <ul>{record[f.field].map((value, i) => (<li key={i}><a onClick={(e) => this.onClick(e, `${f.field}:"${value}"`)} href={`/search?q=${encodeURIComponent(`${f.field}:"${value}"`)}`}>{value}</a></li>))}</ul> });
```

Comment:
I know it may be temporary and that you reverted back, but can you style it so it's on multiple lines and easier to read? ESLint also complains that it exceeds the 100char per line limit.